src/docfx/build/toc/parser/TableOfContentsInputItem.cs

```diff
@@ -19,6 +20,8 @@ internal class TableOfContentsInputItem
 
         public string TocHref { get; set; }
 
+        public JObject Metadata { get; set; }
```

Comment:
>Metadata [](start = 23, length = 8)

per schema, this only exists in root level

Reply:
where is TOC schema? Tianqi told me that leaf metadata will be used for monikers

---
In reply to: [206754936](https://github.com/dotnet/docfx/pull/3141#discussion_r206754936) [](ancestors = 206754936)